src/common/LogClient.cc

```diff
@@ -141,6 +99,50 @@ void LogChannel::set_log_to_monitors(bool v)
   }
 }
 
+void LogChannel::update_config(const clog_targets_conf_t& conf_strings)
```

Comment:
It might be helpful to add a 1-2 sentence comment above each "function overload" pair, such as this one. There are a few pairs like this, but I'd only worry about the pairs (like `update_config`) that have to do with your new implementation. It would be useful for other developers to know which scenario one method is used vs. the other.

Reply:
@ljflores : your comments made me revisit and rewrite my changes. Thanks

Reply:
And re this specific comment: the overloaded version was removed.

Reply:
Cool! I'll take another look.